### PR TITLE
Automation rules: privacy levels

### PR DESCRIPTION
**Pull request by stsewd**
This will be available only for .com

### CODE REVIEW

File: docs/automation-rules.rst [hunk 1]
```diff
@@ -52,14 +52,18 @@ Currently, the following actions are available:
 
 - **Activate version**: Activates and builds the version.
 - **Hide version**: Hides the version. If the version is not active, activates it and builds the version.
-  See more in :ref:`versions:Version States`.
+  See :ref:`versions:Version States`.
+- **Make version public**: Sets the version's privacy level to public.
+  See :ref:`versions:privacy levels`.
+- **Make version private**: Sets the version's privacy level to private.
+  See :ref:`versions:privacy levels`.
```
Comment (humitos) [inline docs/automation-rules.rst:59]: Shouldn't we note here that these are only available in "Read the Docs for Business"?

File: readthedocs/builds/forms.py [hunk 2]
```diff
@@ -131,6 +135,19 @@ def __init__(self, *args, **kwargs):
             (TAG, TAG_TEXT),
         ]
 
+        # Only list supported actions
```
Comment (humitos) [inline readthedocs/builds/forms.py:138]:
```suggestion
        # Remove privacy actions not available in community
```

Adds a little more context about what "supported" means.

File: docs/automation-rules.rst (hunk 1 above)
Comment (stsewd) [reply docs/automation-rules.rst:59]: I thought about that, the linked section already mentions that, so not sure.

Comment (humitos) [reply docs/automation-rules.rst:59]: OK. Maybe not need to repeat that here, then.

**Pull request merged**